js/components/utils.js

```diff
@@ -465,6 +466,21 @@ var Utils = {
     return destination;
   },
 
+  /**
+   * Checks if string is valid
+   *
+   * @function isValidString
+   * @param {String} src - string to be validated
+   * @returns {Boolean} true if string is valid, false if not
+   */
+  isValidString: function(src) {
+    if (src && (typeof src === 'string' || src instanceof String)) {
```

Comment:
consider simplifying this to: `return (src && (typeof src === 'string' || src instanceof String))`